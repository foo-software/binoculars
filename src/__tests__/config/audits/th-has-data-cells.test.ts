import auditThHasDataCells from '../../../config/audits/th-has-data-cells';

describe('th-has-data-cells audit', () => {
  it('should match snapshot', () => {
    expect(auditThHasDataCells('en')).toMatchSnapshot();
  });
});
