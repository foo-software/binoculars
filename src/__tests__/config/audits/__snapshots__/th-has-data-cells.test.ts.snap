// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`th-has-data-cells audit should match snapshot 1`] = `
Object {
  "description": "Screen readers have features to make navigating tables easier. Ensuring table headers always refer to some set of cells may improve the experience for screen reader users. [Learn more](https://web.dev/th-has-data-cells/).",
  "title": "\`<th>\` elements and elements with \`[role=\\"columnheader\\"/\\"rowheader\\"]\` have data cells they describe.",
}
`;
